internal/agent/handlers/sendMetrics.go

```diff
@@ -0,0 +1,35 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/fasdalf/train-go-musthave-metrics/internal/common/constants"
+	"github.com/fasdalf/train-go-musthave-metrics/internal/common/metricstorage"
+	resty "github.com/go-resty/resty/v2"
+)
+
+const URLTemplate = "http://%s/update/"
+
+var address = ``
```

Comment:
Значение по умолчанию для стоки - пустая строка "". Нет необходимости инициализировать ее явно.
 
```suggestion
var address string
```